docs/application/native/guides/connectivity/inm.md

```diff
@@ -0,0 +1,171 @@
+# INM
+
+
+INM means Intelligent Network Monitoring. It can be used to get information about the network provided by well-known linux functions. Some features provided by this API are: detecting IP collision, dumping network status, dumping TCP, monitoring ethernet, monitoring Wi-Fi module state and getting network statistics.
+
+This feature is supported in mobile, tv and wearable profile.
+
+> **Note**  
+> You can test the INM functionality on a target device only. The [emulator](../../../tizen-studio/common-tools/emulator.md) does not support this feature.
+
+## Prerequisites
+
+To enable your application to use the INM API:
+
+1. To use the INM API, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+   ```
+   <privileges>
+      <privilege>http://tizen.org/privilege/network.get</privilege>
+      <privilege>http://tizen.org/privilege/network.set</privilege>
```

Comment:
@suraj-geddam Kindly remove http://tizen.org/privilege/network.set privilege because inm module uses only http://tizen.org/privilege/network.get privilege.

Reply:
Oh. I had put both because it was written that way in the ACR-1250 description, I was not aware that only network.get is sufficient. Done.